releasenotes/README.md

```diff
@@ -155,83 +31,87 @@ Run `sfdx plugins --core` to display the version of Salesforce CLI and the insta
-├─ apex 0.1.0 (core)
-└─ salesforce-alm 50.1.1 (core)
-sfdx-cli 7.76.1 (core)                        // Salesforce CLI version
+└─ salesforce-alm 51.0.1 (core)
+sfdx-cli 7.88.3 (core)                        // Salesforce CLI version
+telemetry 1.1.1 (core)
 ```
 
 ### Uninstall the Pre-Release Version of the salesforcedx CLI Plug-In
 
-If you installed the pre-release version of the `salesforcedx` v50 plug-in, uninstall it and update the CLI. We’re no longer updating the pre-release v50 plug-in.
+If you installed the pre-release version of the `salesforcedx` plug-in, uninstall it and update the CLI. We’re no longer updating the pre-release and will be removing the tag.
 
 ```bash
 $ sfdx plugins:uninstall salesforcedx
 $ sfdx update
 ```
 
+### The Salesforce CLI platform installers now install nodejs v14 LTS
+The platform installers for Salesforce CLI (Windows, Linux and Mac OS X) now install nodejs v14 LTS.
+
+### Salesforce CLI has dropped support for nodejs v8
+Nodejs v8 has reached end-of-life and is no longer maintained. Salesforce CLI supports nodejs engine v10 or greater
+
 ### New Commands
 
-* `force:package:version:displayancestry`
+No new commands have been added in release v51.
 
-    Create visualizations of your package or package version's ancestry tree. View the visualization in Salesforce CLI or use the `--dotcode` parameter to generate output that you can use in graph visualization software.
+### Removed Commands
 
-* `force:package:version:delete`
-* `force:package:delete`
+These commands were removed in v51. The Hammer Test closed pilot program has been terminated.
 
-    Deletes the package or package version.
+* `force:package:hammertest:list`
+* `force:package:hammertest:report`
+* `force:package:hammertest:run`
 
-    You can't recover deleted packages or package versions, or specify them as dependencies. New installations of the package will fail. You can’t delete released versions of second-generation managed packages.
+### New Parameters
 
-### Removed Commands
+* `force:mdapi:deploy --soapdeploy`
 
-These commands were deprecated in v49 and removed in v50.  You can continue using them by installing the [Lightning Testing Service CLI Plug-in](https://github.com/forcedotcom/plugin-lightning-test-service). Salesforce doesn't support or update this plug-in.
+  The `soapdeploy` parameter will cause the metadata deploy to use the SOAP API, instead of the REST API.
 
-* `force:lightning:test:install`
-* `force:lightning:test:run`
+### Deprecated Parameters
 
-### New Parameters
+* `force:org:shape:create`
 
-* `force:package:create --errornotificationusername (-o)`
-* `force:package:update --errornotificationusername (-o)`
+  The `definitionfile` parameter has been removed from this command. This parameter is part of the Org Shape feature which is currently in beta and has been removed based on feedback that users prefer storing Org Shape in the shape source org instead of a scratch definition file.
```

Comment:
Edit of last two sentences: "This parameter is part of the org shape feature which is currently in beta.  We removed the parameter based on feedback from users that they preferred storing the org shape in the shape source org rather than in a scratch definition file."